testing/netatalk/netatalk.initd

```diff
@@ -0,0 +1,21 @@
+#!/sbin/openrc-run
+
+name=netatalk
+command="/usr/sbin/$name"
+command_args="-F $netatalk_config"
+command_background="yes"
+
+extra_started_commands="reload"
```

Comment:
You didn’t write the actual reload function and IIRC there’s no default reload function provided by OpenRC.